src/Microsoft.AspNetCore.Mvc.RazorPages/CompiledPageActionDescriptor.cs

```diff
@@ -30,15 +30,25 @@ public CompiledPageActionDescriptor(PageActionDescriptor actionDescriptor)
         }
 
         /// <summary>
-        /// Gets or sets the <see cref="TypeInfo"/> of the page.
+        /// Gets the list of handler methods for the page. 
         /// </summary>
-        public TypeInfo PageTypeInfo { get; set; }
+        public IList<HandlerMethodDescriptor> HandlerMethods { get; set; }
```

Comment:
I made a variety of updates to these descriptors to make them more consistent with controllers